kfdef/kfctl_aws.yaml

```diff
@@ -350,6 +387,16 @@ spec:
         name: manifests
         path: mpi-job/mpi-operator
     name: mpi-operator
+  - kustomizeConfig:
+      overlays:
```

Comment:
Why do you need cognate overlay?

Reply:
We don't need, copy & paste error here, fixed.